decidim-meetings/spec/system/meeting_spec.rb

```diff
@@ -39,7 +39,7 @@ def visit_meeting
     it "they show it" do
       visit_meeting
 
-      within "#drawer_content_frame" do
+      within "[data-content]" do
```

Comment:
Can we use here a class or other css selector ?

Reply:
Definitely not. That would be a bad practice, as you'd be binding css stylesheets with the testing specs, since the coupling between unrelated techs would increase, something not desiderable.

As long as you can avoid such coupling, through `id` or `data-*`, it is, by far, better. Such attributes belong to the markup, what it's exactly what you're testing in the `*-spec.rb` files, not the css.